examples/cli_example_training_ophys.json

```diff
@@ -0,0 +1 @@
+{"training_params": {"name": "core_trainer", "learning_rate": 0.0001, "loss": "mean_squared_error", "output_dir": "/Users/jeromel/Documents/Work documents/Allen Institute/Projects/Deep2P/repos/public/deepinterpolation/examples", "model_string": "unet_single_1024_mean_squared_error"}, "generator_params": {"steps_per_epoch": 5, "name": "SingleTifGenerator", "pre_post_frame": 30, "train_path": "/Users/jeromel/Documents/Work documents/Allen Institute/Projects/Deep2P/repos/public/deepinterpolation/examples/../sample_data/ophys_tiny_761605196.tif", "batch_size": 5, "start_frame": 0, "end_frame": 100, "pre_post_omission": 0}, "test_generator_params": {"name": "SingleTifGenerator", "pre_post_frame": 30, "train_path": "/Users/jeromel/Documents/Work documents/Allen Institute/Projects/Deep2P/repos/public/deepinterpolation/examples/../sample_data/ophys_tiny_761605196.tif", "batch_size": 5, "start_frame": 0, "end_frame": 100, "pre_post_omission": 1, "steps_per_epoch": -1}, "network_params": {"name": "unet_single_1024"}, "output_full_args": true}
```

Comment:
Can you rewrite this with `json.dumps(...., indent=2)` so that it is more human readable?

Reply:
Done. Good suggestion